src/utils/tests/patches.test.js

```diff
@@ -151,6 +152,7 @@ const podNic = {
 
 const intface = {
```

Comment:
btw, does it have to be called intface ?

Reply:
I don't believe I named this variable, but I'm pretty sure the reason they went with "intface" instead of "interface" is because "interface" is a reserved keyword in JavaScript. I typically use "iface" when shortening it, but it's not much better. Any suggestions?

Reply:
I see, I forgot about that. You can leave it then, if you want..

Reply:
I'll update it in the future if I think of a better alternative.